pkg/kubectl/resource_printer.go

```diff
@@ -2196,6 +2196,10 @@ func printConfigMapList(list *api.ConfigMapList, w io.Writer, options PrintOptio
 func printPodSecurityPolicy(item *extensions.PodSecurityPolicy, w io.Writer, options PrintOptions) error {
 	name := formatResourceName(options.Kind, item.Name, options.WithKind)
 
+	if options.WithNamespace {
```

Comment:
I don't like this pattern.  It's leading to lots of boilerplate and it looks like it is going to make `kubectl get svc ns --all-namespaces` do things I don't like.  

@fabianofranz Do you have a more comprehensive fix in mind?